parcel_test.go

```diff
@@ -28,60 +24,74 @@ func getTestParcel() Parcel {
 	}
 }
 
-// TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
-	// prepare
-	db, err := // настройте подключение к БД
+
+	db, err := sql.Open("sqlite", "tracker.db")
 	store := NewParcelStore(db)
 	parcel := getTestParcel()
-
-	// add
-	// добавьте новую посылку в БД, убедитесь в отсутствии ошибки и наличии идентификатора
-
-	// get
-	// получите только что добавленную посылку, убедитесь в отсутствии ошибки
-	// проверьте, что значения всех полей в полученном объекте совпадают со значениями полей в переменной parcel
-
-	// delete
-	// удалите добавленную посылку, убедитесь в отсутствии ошибки
-	// проверьте, что посылку больше нельзя получить из БД
+	if err != nil {
+		require.NoError(t, err)
+	}
+	defer db.Close()
+
+	numberTest, err := store.Add(parcel)
+	require.NoError(t, err)
+	assert.NotEmpty(t, numberTest)
+
+	posylka, err := store.Get(numberTest)
+	require.NoError(t, err)
+	assert.Equal(t, posylka.Address, parcel.Address)
+	assert.Equal(t, posylka.Client, parcel.Client)
+	assert.Equal(t, posylka.CreatedAt, parcel.CreatedAt)
+	assert.Equal(t, posylka.Status, parcel.Status)
+
+	err1 := store.Delete(numberTest)
+	require.NoError(t, err1)
+	_, err2 := store.Get(numberTest)
```

Comment:
Можно переиспользовать ошибку
```suggestion
	_, err = store.Get(numberTest)
```